app/assets/stylesheets/help.scss

```diff
@@ -1,3 +1,5 @@
+@import "partials/webview";
+
 html, body {
```

Comment:
Slightly unrelated, but related to `help.scss` refactor: I noticed that there's a help screen CSS [class](https://github.com/simpledotorg/simple-server/blob/master/app/assets/stylesheets/application.scss#L49) in `application.scss` – Is that supposed to be there and/or can we move it here?